.conda/meta.yaml

```diff
@@ -69,6 +69,7 @@ requirements:
     - symmetry
     - xlwt
     - dde
+    - scikit-learn
```

Comment:
Would you mind putting this in the right place alphabetically? Also `dde` while you're at it :)

Reply:
fixed